src/Telemetry/AzureDevOpsTimeline/BuildLogScraper.cs

```diff
@@ -16,7 +16,7 @@ public class BuildLogScraper : IBuildLogScraper
     private readonly IClientFactory<IAzureDevOpsClient> _azureDevOpsClientFactory;
 
     // Example: Environment: windows-latest
-    private static readonly Regex _azurePipelinesRegex = new Regex(@"Environment: (\S+)");
+    private static readonly Regex _azurePipelinesRegex = new Regex(@"Image: (\S+)");
```

Comment:
Do you think there's any benefit in checking for both forms? Or are you sure the old "Environment" form is not used at all anymore?

Reply:
The regex didn't catch anything since October, it's probably safe to say the "Environment" form isn't used anymore